build/xtask/src/dist.rs

```diff
@@ -883,15 +898,11 @@ fn update_image_header(
     map: &IndexMap<String, Range<AbiSize>>,
     all_output_sections: &mut BTreeMap<AbiSize, LoadSegment>,
 ) -> Result<bool> {
-    use goblin::container::Container;
     use zerocopy::AsBytes;
 
     let mut file_image = std::fs::read(input)?;
     let elf = goblin::elf::Elf::parse(&file_image)?;
 
-    if elf.header.container()? != Container::Little {
-        bail!("where did you get a big-endian image?");
-    }
```

Comment:
Why was this check disabled, both here and in `load_elf()`?

Reply:
It refers to the whether the elf is 64-bit or not: https://docs.rs/goblin/latest/goblin/container/enum.Container.html